python/pyspark/sql/tests.py

```diff
@@ -4289,14 +4297,15 @@ def data(self):
             .withColumn("v", explode(col('vs'))).drop('vs')
 
     def test_simple(self):
-        from pyspark.sql.functions import pandas_udf, PandasUDFType
-        df = self.data
+        from pyspark.sql.functions import pandas_udf, PandasUDFType, array, col
+        df = self.data.withColumn("arr", array(col("id")))
```

Comment:
minor: It seems a bit arbitrary to mix array type in this test. Array probably belongs to `test_array`, `test_complex_type` sth like `test_all_types`